src/imitation/util/reward_wrapper.py

```diff
@@ -1,11 +1,11 @@
 """Common wrapper for adding custom reward values to an environment."""
 import collections
-from typing import Callable
+from typing import Callable, Optional
 
 import numpy as np
 from stable_baselines.common import vec_env
 
-RewardFn = Callable[[np.ndarray, np.ndarray, np.ndarray, np.ndarray],
+RewardFn = Callable[[np.ndarray, np.ndarray, np.ndarray, Optional[np.ndarray]],
```

Comment:
What are you imagining the semantics being here? Some reward functions ignore steps, but others depend on it and throw an error if not provided?

Reply:
This change is unnecessary, thanks for pointing it out

Reply:
I'm passing in `steps=np.arange(len(obs))` in heatmap reward functions now.